validate/action.yml

```diff
@@ -1,41 +1,48 @@
 name: "Validate"
-description: "Performs the Lula validation and evaluation"
+description: "Performs Lula Validate"
 
 inputs:
-  oscal-target:
-    description: "The target file to be validated"
-  threshold:
-    description: "The assessment result file to compare against. Attempts to find assessment-results.yaml if not specified"
+  component-definition:
+    description: "The component-definition file to be validated"
+  assessment-result:
+    description: "The output location of the assessment result file. Defaults to ./assessment-results.yaml if not specified"
+  target:
+    description: "The framework prop to target"
+  options:
+    description: "Any additional flag options"
 
 runs:
   using: "composite"
   steps:
     - shell: bash
       run: |
-        #!/bin/bash
+       #!/bin/bash
         # Base url for lula artifacts
-        target_dir="${HOME}/.lula-cache"
-        threshold="${{ inputs.threshold }}"
-        target="${{ inputs.oscal-target }}"
+        file_dir="${HOME}/.lula-cache"
+        assessment-result="${{ inputs.assessment-result }}"
+        component-definition="${{ inputs.component-definition }}"
+        target="${{ inputs.target }}"
```

Comment:
Calling out for discussion - is `target` something that the action needs to natively support if `options` exists? 

or rather _should_ it? I don't feel strongly either way but I am looking for how to reduce the amount of conditionals present in both actions.